neo4j-examples/movies-javascript-bolt
# GGP-460: testing with @neo4j-labs/temp-dbs

Proof of concept of using [@neo4j-labs/temp-dbs](https://github.com/neo4j-contrib/neo4j-temp-db) for creating disposable databases for testing.

How to test:

1. Make sure you have a Neo4j database running locally, you can use Neo4j Desktop for this. Make sure to select at least version 4
2. Export the right credentials to your environment like:
```
export NEO4J_URI="bolt://localhost:7687"
export NEO4J_USER="neo4j"
export NEO4J_PASSWORD="123456"
export NEO4J_VERSION="4.2.1"
```
3. Install nodejs dependencies `npm install`
4. Run tests `npm run test`
